docs/topics/kotlin-evolution.md

```diff
@@ -128,9 +128,13 @@ As legacy features get removed and bugs fixed, the source language changes, and
 
 ### Compatibility flags
 
-We provide the -language-version and -api-version flags that make a new version emulate the behavior of an old one, for compatibility purposes. Normally, at least one previous version is supported. This effectively leaves a time span of two full feature release cycles for migration (which usually amounts to about one year). Using an older kotlin-stdlib or kotlin-reflect with a newer compiler without specifying compatibility flags is not recommended, and the compiler will report a [warning](compatibility-modes.md) when this happens.
+We provide the `-language-version X.Y` and `-api-version X.Y` flags that make a new version emulate the behavior of an old
+one for compatibility purposes. To give you more time for migration, we [support](compatibility-modes.md) the development
+for two previous language versions and for three previous API versions in addition to the latest stable one.
```

Comment:
> for two previous language versions

ok, but this may be changed to three sooner than we originally planned, i.e. in 1.6.10/1.6.20

Reply:
thanks, I'll merge this PR and plan KT-49868 for further changes